tobias/utils/motifs.py

```diff
@@ -733,20 +744,33 @@ def get_gimmemotif(self):
 	def get_biomotif(self):
 		""" Get biomotif object for motif """
 
+		# TODO implement
 		self.biomotif_obj = ""
 
 	def get_reverse(self):
 		""" Reverse complement motif """
-		if self.pfm is None:
-			self.get_pfm()
 
-		#Create reverse motif obj
-		reverse_motif = OneMotif()	#empty
-		for att in ["id", "name", "prefix", "strand", "length"]:
-			setattr(reverse_motif, att, getattr(self, att))
+		rev_strand = ["+" if s == "-" else "-" for s in self.strand]

```

Comment:
Again here with the strand - ["+","-"] or ["-","+"] does not make much sense.

Reply:
You are right these two should be considered the same. Again I thought it might be a nice addition to be able to set the strand to unknown.